client/network_connect_test.go

```diff
@@ -87,6 +87,10 @@ func TestNetworkConnect(t *testing.T) {
 				return nil, fmt.Errorf("expected 'container_id', got %s", connect.Container)
 			}
 
+			if connect.EndpointConfig == nil {
```

Comment:
I think `if  connect.EndpointConfig==nil`  then throw exception,never  reach 
`if connect.EndpointConfig.NetworkID != "NetworkID"`，so i add this branch check can be full.